recipes-browser/webkitgtk/webkitgtk_2.36.7.bb

```diff
@@ -106,12 +112,14 @@ ARM_INSTRUCTION_SET:armv7ve = "thumb"
 
 # Install MiniBrowser in PATH
 do_install:append() {
-    if test -f ${D}${libexecdir}/webkit2gtk-4.0/MiniBrowser; then
+    # Execdir is webkit2gtk-4.0 or webkit2gtk-4.1 depends on if USE_SOUP
+    # is or not enabled.
+    if test -f ${D}${libexecdir}/webkit2gtk-4.?/MiniBrowser; then
         mkdir -p ${D}${bindir}
-        mv ${D}${libexecdir}/webkit2gtk-4.0/MiniBrowser ${D}${bindir}
+        mv ${D}${libexecdir}/webkit2gtk-4.?/MiniBrowser ${D}${bindir}
```

Comment:
I know this is unlikely to happen, but using a wildcard means that if both `webkit2gtk-4.0` _and_ `webkit2gtk-4.1` exist in the same destdir, both will be copied over the same binary.

That's why I suggested using a variable, like this:

```python
WEBKITGTK_API_VERSION := "${@bb.utils.contains('PACKAGECONFIG', 'soup2', '4.0', '4.1')}"

# Later...

do_install:append() {
    if test -f "${D}${libexecdir}/webkit2gtk-${WEBKITGTK_API_VERSION}" ; then
        # ...
    fi
}

FILES:${PN} += "${libdir}/webkit2gtk-${WEBKITGTK_API_VERSION}/..."

Reply:
Honestly speaking, I don't know if that expression will be expanded correctly but sounds much better if that works so let's give it a try.

Reply:
Looks like this works. Thanks @aperezdc for your suggestion.

Reply:
Happy to help!